### DISCUSSION TITLE
Core Bridges

### DISCUSSION THREAD

**Discussion by mvalles** [arkmq-org/arkmq-org-broker-operator#693] [Q&A] [answered]
Is there any way to configure core bridges in artemiscloud?
https://activemq.apache.org/components/artemis/documentation/latest/core-bridges.html#configuring-core-bridges

**Comment by gtully** [comment, marked as answer]
yes, via brokerProperties in the CR. 
https://github.com/artemiscloud/activemq-artemis-operator/blob/16f211fa65203b82c4513b3440aaae0abacfdf69/docs/help/operator.md?plain=1#L766

see an example of the format in the test: https://github.com/apache/activemq-artemis/blob/main/artemis-server/src/test/java/org/apache/activemq/artemis/core/config/impl/ConfigurationImplTest.java#L1040

**Reply by mvalles** [reply]
This looks great, @gtully! Thanks.
